ansible/roles/network-interfaces/tasks/main.yml

```diff
@@ -24,4 +24,15 @@
     owner: root
     group: root
     mode: 0644
-  notify: Restart network
+  register: etc_network_interfaces
+
+- name: Restart network
```

Comment:
Please clarify (in a comment in the playbook) why we want this as an inline task rather than a handler (same for the `reload_hostapd` task in the `wifi-ap` role)